src/components/ClaimMasterPanelExt.js

```diff
@@ -40,13 +43,38 @@ class ClaimMasterPanelExt extends Component {
     this.props.clearLastClaimAt();
   }
 
+  getPolicyStatusLabel(timeDelta){
+    let policyStatusLabel = null;
+    if (timeDelta >= 0){
+      policyStatusLabel = "ClaimMasterPanelExt.InsureePolicyEligibilitySummaryActive.header";
+    }
+    else{
+      policyStatusLabel = "ClaimMasterPanelExt.InsureePolicyEligibilitySummaryInactive.header";
+    }
+    return policyStatusLabel;
+  }
```

Comment:
First of all, I recommend creating constants for those long strings, let's say you'll name them ACTIVE_LABEL and INACTIVE_LABEL. Then, it would be best to return it like this:
`
    getPolicyStatusLabel(timeDelta) {
    return timeDelta >= 0 ? ACTIVE_LABEL : INACTIVE_LABEL;
    };
`